lib/plugins/init/index.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const path = require('path');
+
+class Init {
+  constructor(serverless, options) {
+    this.serverless = serverless;
+    this.options = options;
+
+    this.commands = {
```

Comment:
This configuration is very AWS specific, e.g. others might not have the same key, secrert, provider setup. If we build an init command it shouldn't be so provider specific imho